main/src/ca/uwaterloo/flix/language/phase/typer/BaseEffects.scala

```diff
@@ -85,4 +92,100 @@ object BaseEffects {
       Type.mkUnion(tpes, loc)
   }
 
+  /**
+    * Parses a JSON file of the form:
+    *
+    * {{{
+    * {
+    *   "classes": {
+    *     "java.lang.ProcessBuilder": "Exec, FileRead",
+    *     "java.lang.reflect.Method": "Sys"
+    *   }
+    * }
+    * }}}
+    */
+  private def loadClassEffs(): Map[Class[?], Set[Symbol.EffectSym]] = {
+    val data = LocalResource.get(ClassEffsPath)
+    val json = parse(data)
+
+    val m = json \\ "classes" match {
+      case JObject(l) => l.map {
+        case (className, JString(s)) =>
+          val clazz = Class.forName(className)
+          val effSet = s.split(",").map(_.trim).map(Symbol.parseBaseEff).toSet
```

Comment:
Why not use a proper JSON list?

Reply:
But in any case, I think once I or someone starts generating these JSON files it will be more clear what the right format is.